app/src/main/java/com/infomaniak/drive/ui/fileList/multiSelect/MultiSelectActionsBottomSheetDialog.kt

```diff
@@ -157,13 +157,14 @@ abstract class MultiSelectActionsBottomSheetDialog(private val matomoCategory: S
         deletePermanently.isGone = true
     }
 
-    private fun downloadFileArchive() {
-
-        fun downloadArchive(fileIds: IntArray) = liveData(Dispatchers.IO) {
-            emit(ApiRepository.getUUIDArchiveFiles(AccountUtils.currentDriveId, fileIds))
-        }
+    private fun download() {
+        if (navigationArgs.areAllFromTheSameFolder) downloadArchive() else downloadFiles()
```

Comment:
The problem here is that if you download only one file, it will always download an archive. 
Even if it's still weird this action but it would be nice if it was avoided 😅